roles/rkt/files/rkt-gc.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+rkt gc
```

Comment:
would you want to do a `rkt image gc` ?

Reply:
nah that's overkill. we want to clean up the orphaned containers from each time vault/etcd/kubelet restarts